notebooks/import.py

```diff
@@ -171,15 +171,22 @@ def create_collection(dataset_id, metric_id, num_observations, population,
         return cursor.fetchone()[0]
 
 
-def create_point(collection_id, bucket, proportion, count, rank):
+def create_points(collection_id, histogram):
+    params = []
     sql = ('INSERT INTO api_point '
            '(collection_id, bucket, proportion, count, rank) '
-           'VALUES (%s, %s, %s, %s, %s) ')
-    params = [collection_id, bucket, proportion, count, rank]
+           'VALUES %s')
+
+    for rank, kv in enumerate(histogram.iteritems(), 1):
```

Comment:
This is a vestige from the distribution viewer. The rank isn't used in EV and should probably be removed at some point.